src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -171,60 +172,68 @@ docker run <OTHER-DOCKER_OPTIONS> -e CHE_HOST=<ip-addr-or-dns> eclipse/che:<vers
+If you distribute these components on different nodes, hosts or IP addresses, you may need to add additional configuration parameters to bridge the different networks.
 
-Also, since the Che server and your Che workspaces are within containers governed by a Docker daemon, you also need to ensure that these components have good bridges to communicate with the daemon.
+Also, since the Che server and your Che workspaces are within containers governed by a Docker daemon, you must ensure that these components have good bridges to communicate with the daemon.
 
-Generally, if your browser, Che server and Che workspace are all on the same node, then `localhost` configuration will always work.
+Generally, if your browser, the Che server, and the Che workspace are all on the same node, `localhost` configuration will always work.
 
 *WebSockets*
 
-Che relies on web sockets to stream content between workspaces and the browser. We have found many networks and firewalls to block portions of Web socket communications. If there are any initial configuration issues that arise, this is a likely cause of the problem.
+Che relies on WebSockets to stream content between workspaces and the browser. There are many networks and firewalls that block portions of the WebSocket communication. If there are any initial configuration issues that arise, this is a likely cause of the problem.
 
 *Topology*
 
-The Che server runs in its own Docker container, "Che Docker Container", and each workspace gets an embedded runtime which can be a set of additional Docker cotainers, "Docker Container(n)". All containers are managed by a common Docker daemon, "docker-ip", making them siblings of each other. This includes the Che server and its workspaces - each workspace runtime environment has a set of containers that is a sibling to the Che server, not a child.
+The Che server runs in its own Docker container, *Che Docker Container*. Each workspace has an embedded runtime that can be a set of additional Docker cotainers, *Docker Container(n)*. All containers are managed by a common Docker daemon, *docker-ip*, making them siblings of each other. This includes the Che server and its workspaces. Each workspace runtime environment has a set of containers that is a sibling to the Che server and is not a child.
 
 *Connectivity*
 
-The browser client initiates communication with the Che server by connecting to `che-ip`. This IP address must be accessible by your browser clients. Internally, Che runs on Tomcat which is bound to port `8080`. This port can be altered by setting `CHE_PORT` during start or in your `che.env`.
+The browser client initiates communication with the Che server by connecting to the `che-ip`. This IP address must be accessible by your browser clients. Internally, Che runs on Tomcat that is bound to the port `8080`. This port can be altered by setting the `CHE_PORT` variable during start or in the `che.env` file.
```

Comment:
"Connecting to the `che-ip`" what? There's a noun missing here.

The original wording with "bound to port `8080`" was fine here, too.